locales/es.json

```diff
@@ -36,6 +36,10 @@
                 "title": "Explorador de bloques witnet.network",
                 "content": "El explorador de bloques witnet.network te da una visión neutral de lo que está sucediendo en la blockchain de Witnet."
             },
+            "data_feed_explorer": {
+                "title": "Witnet Data Feed Explorer",
+                "content": "El explorador muestra los datos publicos recogidos por la red de Witnet a tiempo real y reflejados en las diferentes redes."
+            },
```

Comment:
Let's please always deal with translations on Crowdin, otherwise, it's just too easy to end up with inconsistent translations and PR conflicts!